practice/1_python_part_1/task3.py

```diff
@@ -17,12 +17,12 @@
 
 def build_from_unique_words(*lines: Iterable[str], word_number: int) -> str:
 
-    words = []
+    words = ''
     for line in lines:
 
         line_words = sorted(list(set(line.split())))
         if word_number < len(line_words):
-            words.append(line_words[word_number])
+            words+=line_words[word_number] + " "
```

Comment:
It is more convenient to work with list that with string. You could just make a string at the very end by " ".join(words)